src/commodore/index.spec.ts

```diff
@@ -1,18 +1,49 @@
-import { loadFixture } from '../test/util';
-import { extractPackageFile } from './index';
+import { mkdir } from 'fs/promises';
+import { getFixturePath, loadFixture } from '../test/util';
+import { extractPackageFile, defaultConfig } from './index';
+import { clearCache } from './inventory';
 import { expect, describe, it } from '@jest/globals';
```

Comment:
```suggestion
import { jest, beforeEach, beforeAll, expect, describe, it } from '@jest/globals';
```
You actually use these. Not sure how this works without the imports.